mod/projects/set/self/project/core.haml

```diff
@@ -0,0 +1,14 @@
+%div
+  = field_nest :description
+  = nest :homepage_featured_projects, items: { view: "listing" }
+#project-tabs.tabbable
+  %ul.nav.nav-tabs
+    %li.nav-item
+      %a{class: "active nav-link", href: "#project-active", data: {toggle: "tab"}} Active
+    %li.nav-item
+      %a{class: "nav-link", href: "#project-inactive", data: {toggle: "tab"}} Inactive
+  .tab-content
+    #project-active.tab-pane.active
```

Comment:
as I've said a couple of times, this should loop through all options cards.

Reply:
Here's what I want you to do.

Warning: some of this is fun ;)

1. using staging or dev, do some sharking, and make it so that any `[Status]+Projects` card will produce a search for projects with that status.  For example, `Active+Projects` will give you the same thing that `Project+Active` currently does.  You do this by creating a `*structure` rule that applies to `[Status]+Projects`.  You will need to use WQL syntax.  We use that all the time, so this will be good practice.
2. Once you have that working, I want you to add all the cards you used to make this work to the "export" card.
3. Then you can use `card:pull` to pull down the contents of the export card to a migration. This will make it so that when we deploy live, it will add all those cards.
4. Then, in the ruby code, you will need to get a list of all the status options for projects.  See `https://wikirate.org/Project+status+*type_plus_right+*options`.  I think I've already added a comment somewhere discussing this more.
5. Then in the HAML, you will need to loop through each of those options, and then nest the `+:project` card in it.

If you do this, then all someone has to do is add a new Status to the options list (which will happen soon), and then the interface will handle it all without any additional coding.